src/V3Ast.h

```diff
@@ -2491,11 +2493,11 @@ class AstNode VL_NOT_FINAL {
         foreachImpl<const T_Node>(this, f, /* visitNext: */ true);
     }
 
-    // Given a predicate 'p' that takes a single argument of some AstNode subtype 'T_Node', return
-    // true if and only if there exists a node of type 'T_Node' in the tree rooted at this node,
-    // that satisfies the predicate 'p'. Returns false if no node of type 'T_Node' is present.
-    // Traversal is performed in some arbitrary order and is terminated as soon as the result can
-    // be determined.
+    // Given a predicate 'p' that takes a single argument of some AstNode subtype 'T_Node',
+    // return true if and only if there exists a node of type 'T_Node' in the tree rooted at
+    // this node, that satisfies the predicate 'p'. Returns false if no node of type 'T_Node'
+    // is present. Traversal is performed in some arbitrary order and is terminated as soon as
+    // the result can be determined.
```

Comment:
This I have to investigate... it should not be, but I'll look into it and solve this!

Reply:
Unfortunately clang-format does not format comments beyond enforcing the line length, so the CI can't revert this. I tend to get these because I have auto-format-on-save in my editor, but if I save with a missing closing brace it will reindent the rest of the file, then breaks the comments to fix the line lengths. When you fix the closing brace, the comment lines are already the right length, so it won't undo the change. Just need to manually revert the unnecessary changes.

Reply:
Ok, I guess I can close it then, I reverted it!
Thank you!